sql/analyzer/resolve_insert_literals.go

```diff
@@ -0,0 +1,82 @@
+package analyzer
+
+import (
+	"github.com/src-d/go-mysql-server/sql"
+  "github.com/src-d/go-mysql-server/sql/expression"
+	"github.com/src-d/go-mysql-server/sql/plan"
+  errors "gopkg.in/src-d/go-errors.v1"
+
+)
+
+var errWrongNumberOfValues = errors.NewKind("the number of values to insert differ from the expected columns")
+
+func convertIntegerLiteralsInsert(ctx *sql.Context, analyzer *Analyzer, originalNode sql.Node) (sql.Node, error) {
+  span, _ := ctx.Span("resolve_insert_literals")
+  defer span.Finish()
+
+  return plan.TransformUp(originalNode, func(node sql.Node) (sql.Node, error) {
+
```

Comment:
leftover blank line

Reply:
Done in https://github.com/src-d/go-mysql-server/pull/840/commits/6d4af9b32ba3d74f35ab0d518029332cb797ceac